src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATE_HISTORY = [];
```

Comment:
It's not constant in general as you add values to it, so it should be in camelCase